xbmc/addons/kodi-addon-dev-kit/include/kodi/kodi_vfs_types.h

```diff
@@ -41,7 +41,7 @@ extern "C"
     char* path;              //!< item path
     int num_props;           //!< Number of properties attached to item
     VFSProperty* properties; //!< Properties
-    //FILETIME mtime;          //!< Mtime for file represented by item
+    time_t date_time;        //!< Mtime for file represented by item
```

Comment:
@notspiff : Are you fine with the change from FILETIME to time_t here? It's obviously a change of the interface.

Reply:
Not really, since there was no interface before.

Reply:
Maybe include the clock in the comment, like "in Unix time"